src/App.js

```diff
@@ -34,39 +35,32 @@ const store = createStore(
 );
 
 /* eslint-disable camelcase */
+export const DetailComponents = () => {
+  const { id } = useParams();
 
-export class DetailComponents extends React.Component {
-  render() {
-    const complaint_id = this.props.match.params.id;
-
-    return (
-      <IntlProvider locale="en">
-        <main role="main">
-          <ComplaintDetail complaint_id={complaint_id} />
-        </main>
-      </IntlProvider>
-    );
-  }
-}
-
-DetailComponents.propTypes = {
-  match: PropTypes.object.isRequired,
+  return (
+    <IntlProvider locale="en">
+      <main role="main">
+        <ComplaintDetail complaint_id={id} />
+      </main>
+    </IntlProvider>
+  );
 };
-
 /* eslint-enable camelcase */
 
 // eslint-disable-next-line react/no-multi-comp
-export class App extends React.Component {
-  render() {
-    return (
-      <Provider store={store}>
-        <Router>
-          <Switch>
-            <Route path="*/detail/:id" component={DetailComponents} />
-            <Route path="/" component={SearchComponents} />
-          </Switch>
-        </Router>
-      </Provider>
-    );
-  }
-}
+/**
```

Comment:
Do we need a JSDoc here?

Reply:
Yes. 

https://github.com/cfpb/ccdb5-ui/pull/425/commits/30adbeebc8e4c3cd9054aa5f6b85bd40e3380ff1